pos_pricelist/static/src/js/models.js

```diff
@@ -513,6 +513,11 @@ function pos_pricelist_models(instance, module) {
                 }
                 break;
             }
+            //default price_extra is not included (product variants)
+            if(product['price_extra'] > 0.0){
+                price += product['price_extra']
+            }
+            
```

Comment:
Sorry for these changes, I meant them for another PR but I merged with this one. My mistake.
I have already deleted them.